signup-service/src/main/java/io/mosip/signup/dto/RegisterRequest.java

```diff
@@ -25,4 +25,6 @@ public class RegisterRequest {
 
     @NotNull(message = ErrorConstants.INVALID_USERINFO)
     private @Valid UserInfoMap userInfo;
+
+    private String locale;
```

Comment:
Do we need validation only if a value is provided in this field? I believe we have language validator annotation already with a required boolean flag @mengleang-0090